MapboxDirections/MBDirections.swift

```diff
@@ -103,7 +103,7 @@ open class Directions: NSObject {
         If the request was canceled or there was an error obtaining the routes, this parameter is `nil`. This is not to be confused with the situation in which no results were found, in which case the array is present but empty.
      - parameter error: The error that occurred, or `nil` if the placemarks were obtained successfully.
      */
-    public typealias RouteCompletionHandler = (_ waypoints: [Waypoint]?, _ routes: [Route]?, _ error: NSError?) -> Void
+    public typealias RouteCompletionHandler = (_ waypoints: [Waypoint]?, _ routes: [Route]?, _ JSONRoutes: [[String: Any]]?, _ error: NSError?) -> Void
```

Comment:
Is there any way we could make this a non breaking change? What if JSONRoutes was nil by default?

Reply:
That would still be a breaking change. There’s no way to add an argument to this closure without breaking backwards compatibility.